.github/workflows/ci-build.yaml

```diff
@@ -236,6 +236,8 @@ jobs:
     needs: build
     if: needs.build.outputs.packages_were_built == 'true'
 
+    timeout-minutes: 60
```

Comment:
This seems high?

Reply:
yes, I was wondering that when I open the PR, maybe 30 min max?